docs/integrations/app-integrations/pypi.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: PyPI
+sidebarTitle: PyPI
+---
+
+In this section, we present how to connect PyPI to MindsDB.
+
+[PyPI](https://pypi.org) is a host for maintaining and storing Python packages. It's a good place for publishing your Python packages in different versions and releases.
+
+Data from PyPI can be utilized within MindsDB to train models and make predictions about your Python packages.
+
+## Connection
+
+This handler is implemented using the standard Python `requests` library. It is used to connect to the RESTful service that [pypistats.org](https://pypistats.org) is serving.
+
+There are no connection arguments required to initialize the handler.
+
+To connect to PyPI using MindsDB, the following CREATE DATABASE statement can be used:
+
+```sql
+CREATE DATABASE mediawiki_datasource
```

Comment:
Let's call it `pypi_datasource`.

Reply:
Oh, my bad, @martyna-mindsdb. It's done.